README.md

```diff
@@ -23,9 +23,19 @@ The config is in [ewasm-testnet-cpp-config.json](ewasm-testnet-cpp-config.json).
 
 Example node with mining on single CPU core, with no bootstrap:
 
+1) Build (hera)[https://github.com/ewasm/hera] as a shared library:
+```
+git clone https://github.com/ewasm/hera --recursive
+cd hera
+cmake -DBUILD_SHARED_LIBS=ON .
```

Comment:
Two dots...

Reply:
But I would suggest to copy it from the geth section:
```
 	mkdir build && cd build
 	cmake .. -DBUILD_SHARED_LIBS=ON
 	cmake --build .
```

Reply:
Fixed now.

Reply:
Done